src/main/kotlin/no/nav/syfo/infrastructure/cronjob/RepublishForhandsvarselWithAdditionalInfoCronjob.kt

```diff
@@ -0,0 +1,104 @@
+package no.nav.syfo.infrastructure.cronjob
+
+import no.nav.syfo.application.service.VurderingService
+import no.nav.syfo.domain.DocumentComponent
+import no.nav.syfo.domain.DocumentComponentType
+import no.nav.syfo.domain.Vurdering
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+import java.util.*
+
+class RepublishForhandsvarselWithAdditionalInfoCronjob(
+    private val vurderingService: VurderingService,
+    private val uuids: List<String> = emptyList(),
+) : Cronjob {
+    override val initialDelayMinutes: Long = 4
+    override val intervalDelayMinutes: Long = 10000000
+
+    private val log = LoggerFactory.getLogger(RepublishForhandsvarselWithAdditionalInfoCronjob::class.java)
+
+    override suspend fun run(): List<Result<Vurdering>> {
+        val newFrist = LocalDate.of(2025, 4, 7) // 7. april 2025
+        val result = uuids.map { uuid ->
+            try {
+                val vurdering = vurderingService.getVurdering(UUID.fromString(uuid))
+                if (vurdering != null) {
+                    val newDocument = generateNewDocument(vurdering, newFrist)
+                    val newVurdering = vurderingService.createVurdering(
+                        personident = vurdering.personident,
+                        veilederident = vurdering.veilederident,
+                        type = vurdering.type,
+                        arsak = vurdering.arsak,
+                        begrunnelse = vurdering.begrunnelse,
+                        document = newDocument,
+                        gjelderFom = vurdering.gjelderFom,
+                        svarfrist = newFrist,
+                        callId = "cronjob-republish-forhandsvarsel",
+                        overrideForhandsvarselChecks = true,
+                    )
+                    Result.success(newVurdering)
+                } else {
+                    Result.failure(IllegalArgumentException("Vurdering with UUID $uuid not found"))
+                }
+            } catch (e: Exception) {
+                log.error("Exception caught while attempting to republish forhandsvarsel with UUID $uuid", e)
+                Result.failure(e)
+            }
+        }
+        log.info("""
+            Updated ${result.count { it.isSuccess }} forhandsvarsler in ${RepublishForhandsvarselWithAdditionalInfoCronjob::class.java.simpleName}.
+            UUIDs for new forhandsvarsler: ${result.filter { it.isSuccess }.joinToString(", ") { it.getOrNull()?.uuid.toString() }}
+        """.trimIndent())
```

Comment:
For å kunne få ut UUIDene i logger, feks. Kan sikkert løses på andre måter også.